src/cdomains/baseDomain.ml

```diff
@@ -6,20 +6,14 @@ module BI = IntOps.BigIntOps
 
 module CPA =
 struct
+  module M0 = MapDomain.MapBot (Basetype.Variables) (VD)
```

Comment:
Why did you remove `HashCashed` here?

Reply:
It's still there below on line 15. This is just the innermost `MapDomain` whose printing is changed to preserve the old grouping. After that it's wrapped the same way.

Reply:
But the wrapping in line 15 only applies to `Glob`, not the `CPA` itself. Or am I missing something?

Reply:
ah, nvm the diff is just a bit confusing. sorry about that

Reply:
There's no `Glob`, it was just some old unused module that I removed.

Reply:
Yeah, sorry. It's good to merge then.